openwisp_notifications/settings.py

```diff
@@ -20,9 +21,31 @@
     settings, 'OPENWISP_NOTIFICATION_HTML_EMAIL', True
 )
 
+OPENWISP_NOTIFICATION_HOST = getattr(settings, 'OPENWISP_NOTIFICATION_HOST', None)
+OPENWISP_NOTIFICATION_SOUND = getattr(
+    settings,
+    'OPENWISP_NOTIFICATION_SOUND',
+    '/static/openwisp_notifications/audio/notification_bell.mp3',
+)
+
 
 def get_config():
     user_config = getattr(settings, 'OPENWISP_NOTIFICATIONS_CONFIG', {})
     config = CONFIG_DEFAULTS.copy()
     config.update(user_config)
     return config
+
+
+def validate_notification_host_configuration():
+    if OPENWISP_NOTIFICATION_HOST:
+        if not (
+            'corsheaders' in settings.INSTALLED_APPS
+            and 'corsheaders.middleware.CorsMiddleware' in settings.MIDDLEWARE
+        ):
+            raise ImproperlyConfigured(
```

Comment:
I don't remember who started first, but somebody used django system checks recently to do this and the result is very good:
https://docs.djangoproject.com/en/3.0/topics/checks/

May have been @nepython or @atb00ker, I don't remember, but maybe we should start adopting this method since it's designed specifically for this purpose and achieves a very good result.